framework/src/actions/SetupMeshAction.C

```diff
@@ -208,14 +215,14 @@ SetupMeshAction::act()
     }
 
     _mesh = _factory.create<MooseMesh>(_type, "mesh", _moose_object_pars);
-    if (isParamValid("displacements"))
+    if (isParamValid("displacements") && (getParam<bool>("use_displaced_mesh") == true))
       _displaced_mesh = _factory.create<MooseMesh>(_type, "displaced_mesh", _moose_object_pars);
   }
   else if (_current_task == "init_mesh")
   {
     _mesh->init();
 
-    if (isParamValid("displacements"))
+    if (isParamValid("displacements") && (getParam<bool>("use_displaced_mesh") == true))
```

Comment:
Did you want to change this to
```
if (isParamValid("displacements") && getParam<bool>("use_displaced_mesh"))
```
?

Reply:
Yes -- just getting to it right this second, coincidentally.

Reply:
Ok, I just changed those.